src/applications/static-pages/homepage-banner/HomepageBanner.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import * as Sentry from '@sentry/browser';
+import classnames from 'classnames';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import sanitizeHtml from 'sanitize-html';
+import yaml from 'js-yaml';
+import localStorage from 'platform/utilities/storage/localStorage';
+
+const VAGOV_CONTENT =
+  'https://raw.githubusercontent.com/department-of-veterans-affairs/vagov-content/master';
+
+const HOMEPAGE_BANNER_YML_LOCATION = `${VAGOV_CONTENT}/fragments/home/banner.yml?cache-break=${new Date().getTime()}`;
+
+const ACCEPTABLE_CONTENT_TAGS = [
+  'h4',
+  'h5',
+  'h6',
+  'blockquote',
+  'p',
+  'a',
+  'ul',
+  'ol',
+  'li',
+  'b',
+  'i',
+  'strong',
+  'em',
+  'hr',
+  'br',
+  'div',
+];
```

Comment:
No `'span'`s?

Reply:
Spans are actually considered a security risk

Reply:
Just kidding, it just didn't seem like an element that would be used in the homepage banner so it didn't occur to me. I'll add it!